app/main.py

```diff
@@ -1,8 +1,39 @@
+from __future__ import annotations
```

Comment:
Due to the constraints of the tests, remove this import from the file, class `Person` should be placed first.